_includes/figure

```diff
@@ -5,6 +5,9 @@
     {% else %}
       "{{ include.image_path }}"
     {% endif %}
+    {% if include.image_path contains ".svg" %}
```

Comment:
is this for an unrelated bugfix?

Reply:
Oops that shouldn't be in there :x but yes, unrelated bugfix